src/Traits/OneSignalHelpers.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
I think you are probably better off just `use`ing these in the `OneSignalMessage` class rather than using a trait that is only there to use other traits IMO